tests/t520-operator.c

```diff
@@ -168,6 +170,16 @@ int main(int argc, char **argv) {
   CeedCompositeOperatorAddSub(op_mass, op_mass_tet);
   CeedCompositeOperatorAddSub(op_mass, op_mass_hex);
 
+  {  // Test CeedCompositeOperatorGetSubByName
+    CeedOperator op_byname;
+
+    CeedCompositeOperatorGetSubByName(op_mass, "mass_hex", &op_byname);
```

Comment:
```suggestion
    CeedCompositeOperatorGetSubByName(op_mass, "mass hex", &op_byname);
```
and this?

Reply:
Whoops. Helps if the test itself also passes...